.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Updated .ocamlformat to use the latest version
+commit hash: a6a668fc2780307116001f76b07a3f1a1be02592
```

Comment:
```suggestion
a6a668fc2780307116001f76b07a3f1a1be02592
```

The lines in this file should just be commit hashes. You can see an example of such a file used to ignore formatting changes in ppxlib here: https://github.com/ocaml-ppx/ppxlib/blob/main/.git-blame-ignore-revs.